src/auth/auth.controller.ts

```diff
@@ -23,4 +26,10 @@ export class AuthController {
   login(@Body() loginDto: LoginDto): Promise<AuthResponseDto> {
     return this.authService.login(loginDto);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @Get("me")
```

Comment:
Any reason why this is in the auth controller?
I believe this was in the users controller in the template

Reply:
Yes bhaiya it was, as I was registering my user's in the auth, so I thought of fetching the user from the auth too.

Reply:
This should be from the users controller.